tasks/build.js

```diff
@@ -16,7 +17,12 @@ module.exports = function(grunt) {
     combined = combined.replace(/GENERATED_CDN_VSN/g, grunt.vjsVersion.majorMinor);
```

Comment:
when we get browserify, there's a transform that does this automatically for us.